### PR TITLE
Run3-gex98A Use of ESGetToken in Geometry/TrackerNumberingBuilder

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Use of ESGetToken in Geometry/TrackerNumberingBuilder

#### PR validation:

Use the runTheMatrix test workflows

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

File: Geometry/TrackerNumberingBuilder/test/GeometricDetAnalyzer.cc [hunk 1]
```diff
@@ -25,7 +25,6 @@
 
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/EventSetup.h"
-#include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline Geometry/TrackerNumberingBuilder/test/GeometricDetAnalyzer.cc:28]: `pDD` below is a `ESHandle`: therefore this should be maintained

File: Geometry/TrackerNumberingBuilder/test/ModuleNumbering.cc [hunk 2]
```diff
@@ -22,7 +22,6 @@
 
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/EventSetup.h"
-#include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline Geometry/TrackerNumberingBuilder/test/ModuleNumbering.cc:25]: `rDD` below is a `ESHandle`: therefore this should be maintained

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

was already approved by geometry

**Comment by qliphy** [comment]
+1

**Pull request merged**